api/Request.json

```diff
@@ -32,17 +65,28 @@
             "version_added": "28"
           },
           "safari": {
-            "version_added": false
+            "version_added": "10.1"
           },
           "safari_ios": {
-            "version_added": false
+            "version_added": "10.1"
           },
           "samsunginternet_android": {
             "version_added": "4.0"
           },
-          "webview_android": {
-            "version_added": "42"
-          }
+          "webview_android": [
+            {
+              "version_added": "42"
+            },
+            {
+              "version_added": "41",
```

Comment:
> IIRC, WebView doesn't allow preferences. We probably only need the "42" version data here.

This is true. I'm trying to find someplace where we say that explicitly. I'd like to add a link to BCD instructions so that I don't need to keep saying that.

Reply:
@jpmedley if you do find something like that, can you open an issue about it? I think it would be pretty straightforward to have the linter disallow prefs in webview data.